util/unifyfs-stage/src/unifyfs-stage-transfer.c

```diff
@@ -0,0 +1,376 @@
+/*
+ * Copyright (c) 2020, Lawrence Livermore National Security, LLC.
+ * Produced at the Lawrence Livermore National Laboratory.
+ *
+ * Copyright 2020, UT-Battelle, LLC.
+ *
+ * LLNL-CODE-741539
+ * All rights reserved.
+ *
+ * This is the license for UnifyFS.
+ * For details, see https://github.com/LLNL/UnifyFS.
+ * Please read https://github.com/LLNL/UnifyFS/LICENSE for full license text.
+ */
+#include <config.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+#include <errno.h>
+#include <ctype.h>
+#include <limits.h>
+#include <sys/time.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <dirent.h>
+#include <libgen.h>
+#include <getopt.h>
+#include <time.h>
+#include <mpi.h>
+#include <openssl/md5.h>
+
+#include "unifyfs-stage.h"
+
+static int md5_checksum(const char* path, unsigned char* digest)
+{
+    int ret = 0;
+    size_t len = 0;
+    int fd = -1;
+    unsigned char data[1024] = { 0, };
```

Comment:
This doesn't need to be initialized to zero

Reply:
I removed all these.